services/core-api/app/api/verifiable_credentials/resources/verifiable_credential_map.py

```diff
@@ -0,0 +1,90 @@
+            'party_guid',
+            type=str,
+            help='GUID of the party.',
+            location='json',
+            store_missing=False)
+    parser.add_argument(
+        'permit_amendment_guid', location='json', type=str, store_missing=False)
+    
+
+    @api.doc(description="Create a connection invitation for a party by guid", params={"party_guid":"guid for party with wallet connection","permit_amendment_guid":"parmit_amendment that will be offered as a credential to the indicated party"})
+    def post(self):
+
+        data = self.parser.parse_args()
+        current_app.logger.warning(data)
+        party_guid = data["party_guid"]
+        permit_amendment_guid = data["permit_amendment_guid"]
+
+        # validate action
+        party = Party.find_by_party_guid(party_guid)
+        if not party:
+            raise BadRequest(f"party not found with party_guid {party_guid}")
+
+
+        permit_amendment = PermitAmendment.find_by_permit_amendment_guid(permit_amendment_guid)
+        if not (permit_amendment):
+            raise BadRequest(f"permit_amendment not found")
+
+        # collect information
+        # https://github.com/bcgov/bc-vcpedia/blob/main/credentials/credential-bc-mines-act-permit.md#261-schema-definition
+        credential_attrs={}
+
+        credential_attrs["permit_no"] = permit_amendment.permit_no
+        credential_attrs["permit_status_code"] = permit_amendment.permit.permit_status_code
+        credential_attrs["mine_party_appt"] = permit_amendment.permit.current_permittee
+        credential_attrs["mine_operation_status_code"] = permit_amendment.mine.mine_status[0].mine_status_xref.mine_operation_status_code
+        credential_attrs["mine_operation_status_reason_code"] = permit_amendment.mine.mine_status[0].mine_status_xref.mine_operation_status_reason_code
+        credential_attrs["mine_operation_status_sub_reason_code"] =  permit_amendment.mine.mine_status[0].mine_status_xref.mine_operation_status_sub_reason_code
+        credential_attrs["mine_commodity_code"] = permit_amendment.mine.mine_type[0].mine_type_detail[0].mine_commodity_code
+        credential_attrs["mine_disturbance_code"] = permit_amendment.mine.mine_type[0].mine_type_detail[0].mine_disturbance_code
+        credential_attrs["mine_no"] = permit_amendment.mine.mine_no
+        credential_attrs["issue_date"] = permit_amendment.issue_date
+        credential_attrs["latitude"] = permit_amendment.mine.latitude
+        credential_attrs["longitude"] = permit_amendment.mine.longitude
+        credential_attrs["bond_total"] = permit_amendment.permit.active_bond_total
+        credential_attrs["tsf_operation_count"] = len([tsf for tsf in permit_amendment.mine.mine_tailings_storage_facilities if tsf.tsf_operating_status_code == "OPT"])
+        credential_attrs["tsf_care_and_maintainence_count"] = len([tsf for tsf in permit_amendment.mine.mine_tailings_storage_facilities if tsf.tsf_operating_status_code == "CAM"])
+
```

Comment:
This is defined by the governance document commented directly above. 

The values I believe are the intended ones.